doc/source/configuration/external_satpy.rst

```diff
@@ -23,32 +23,40 @@ It is your responsibility to make sure the setting points to a suitable Satpy
 package: If the given path doesn't point to a Python package directory or not to
 one providing Satpy, the application may exit immediately throwing Exceptions.
 
-Using Extra Readers
-===================
+Using Extra Satpy component configuration
+=========================================
+
+SIFT can use external `Satpy component configuration <https://satpy.readthedocs.io/en/stable/config.html#component-configuration>`_ folder,
+that hosts extra ``readers``, ``composites``, ``enhancements`` and ``areas`` definitions.
+To use external satpy component configuration it is necessary to define
+``satpy_extra_config_path`` in `external_satpy.yaml <https://github.com/ssec/sift/blob/master/uwsift/etc/SIFT/config/external_satpy.yaml>`_::
+
+    satpy_extra_config_path: [directory path]
+
+or it can be defined via environmet variable ``SATPY_CONFIG_PATH`` as described `here <https://satpy.readthedocs.io/en/stable/config.html#config-path-setting>`_.
```

Comment:
Thanks for your changes, but before a noticed this suggestion I pushed f7287c405c5490d80b1abde391a9b46e1f4f02a4 and (because of it?) I could not commit your suggestion. So I made 180386c1e198c88ba45ee5cc8d12ebac9ad3bc05 to include your version. Sorry for the mess.